src/runtime/Makefile

```diff
@@ -139,7 +139,8 @@ HYPERVISORS := $(HYPERVISOR_ACRN) $(HYPERVISOR_FC) $(HYPERVISOR_QEMU) $(HYPERVIS
 QEMUPATH := $(QEMUBINDIR)/$(QEMUCMD)
 QEMUVALIDHYPERVISORPATHS := [\"$(QEMUPATH)\"]
 
-QEMUVALIDVIRTIOFSPATHS := $(QEMUBINDIR)/$(QEMUVIRTIOFSCMD)
+QEMUVIRTIOFSPATH := $(QEMUBINDIR)/$(QEMUVIRTIOFSCMD)
```

Comment:
@c3d, quck question.

RHEL, CentOS, and Fedora do ship virtiofsd on LIBEXECDIR. Do we know whether other distros do the same? Where do we install using kata-deploy?

Asking because depending on the "default" location expected,  may be worth to change QEMUBINDIR to LIBEXECDIR.

Reply:
OK with the idea, but I would move that to a separate PR. Here, I'm just trying to repair some damage.

Reply:
Ack! We can file a new issue and track that separately.